locks.go

```diff
@@ -17,6 +17,8 @@ type DistLocker interface {
 	Lock(ctx context.Context) error
 	// Unlock unlocks the previously successfully locked lock.
 	Unlock(ctx context.Context) error
+	// Name of the lock
+	Name() string
```

Comment:
If this name is used merely for tests then it should not be a part of this public interface.
Instead, this name should be defined in tests: may be in `distLockers()`?

Reply:
Okay, add name to `distLockers()` and `lockers()` instead.